### PR TITLE
[SDKIOS-908] Widget made alive

### PR DESCRIPTION
**Pull request by szymon-onewelcome**

### CODE REVIEW

File: widget/ResourceGateway.swift [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
-                  let responseJsonData = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: String] else {
-                      completion(nil)
-                      return
-                  }
-            let userIdDecorated = responseJsonData["decorated_user_id"]
-            completion(userIdDecorated)
-        }
-    }
-}
+////  Copyright © 2020 Onegini. All rights reserved.
```
Comment (kfa-onewelcome) [inline widget/ResourceGateway.swift:1]: why this is all commented?

File: widget/Startup.swift [hunk 2]
```diff
@@ -1,4 +1,4 @@
-//  Copyright © 2020 Onegini. All rights reserved.
+//  Copyright © 2020 OneWelcome. All rights reserved.
 
 import OneginiSDKiOS
```
Comment (kfa-onewelcome) [inline widget/Startup.swift:3]: how about the warning I mentioned, that the sdk cannot be used in the widget?

File: widget/ResourceGateway.swift (hunk 1 above)
Comment (szymon-onewelcome) [reply widget/ResourceGateway.swift:1]: The file will be removed.

File: widget/Startup.swift (hunk 2 above)
Comment (szymon-onewelcome) [reply widget/Startup.swift:3]: I checked locally and I think it is on SDK side and this tick should do the trick

File: widget/ResourceGateway.swift [hunk 3]
```diff
@@ -1,41 +1,41 @@
...
+//            guard let data = response?.data,
+//                  let responseJsonData = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: String] else {
+//                      completion(nil)
+//                      return
+//                  }
+//            let userIdDecorated = responseJsonData["decorated_user_id"]
+//            completion(userIdDecorated)
+//        }
+//    }
+//}
```
Comment (kfa-onewelcome) [inline widget/ResourceGateway.swift:41]: can you remove the commented code here?

**Review by kfa-onewelcome — APPROVED** [review]

**Pull request merged**